include/robot_interfaces/pybind_camera.hpp

```diff
@@ -0,0 +1,69 @@
+#include <pybind11/eigen.h>
+#include <pybind11/pybind11.h>
+#include <pybind11/stl_bind.h>
+
+#include <opencv2/opencv.hpp>
+#include <robot_interfaces/sensor_backend.hpp>
+#include <robot_interfaces/sensor_data.hpp>
+#include <robot_interfaces/sensor_data_types.hpp>
+#include <robot_interfaces/sensor_frontend.hpp>
+
+namespace robot_interfaces
+{
+/**
+ * @brief Create python bindings for different camera types.
+ *
+ * @tparam An instance of SensorDataTypes
+ */
+
+template <typename Types>
+void create_camera_bindings(pybind11::module& m)
+{
+    pybind11::class_<typename Types::Data, typename Types::DataPtr>(
+        m, "SensorData")
+        .def(pybind11::init<>());
+
+    pybind11::class_<typename Types::Backend>(m, "SensorBackend")
```

Comment:
To be a bit more compact and consistent with the robot types, remove the "Sensor" from the Python type.  It will be clear from the namespace what kind of backend it is (e.g. we have `robot_interfaces.finger.Backend` and `robot_interfaces.trifinger.Backend`).